Add alternative make_line function with an example (#4470)
qgis/QGIS-Documentation: source/docs/user_manual/working_with_vector/expression.rst

    CASE WHEN $area > 10 000 THEN 'Larger' ELSE 'Smaller' END

* You can manipulate the current geometry with the variable $geometry to create

>   a buffer or get the point on surface::

Author: I thought "point on surface" was a particular point and not simply a point within the geometry

>   a buffer or get the point on surface::

Reviewer: The position of the "point on surface" depends on the algorithm used. In this way it is different from the centroid, which is well defined.

    buffer( $geometry, 10 )
    point_on_surface( $geometry )

* Given a point feature, generate a closed line to whom it's the center point::

    make_line(
      -- using an array of points placed around the original
      array_foreach(

>         -- list of angles to place a projected point (every 90°)

Reviewer: ```suggestion
        -- list of angles for placing the projected points (every 90°)
```

        array:=generate_series( 0, 360, 90 ),

>         -- translate point for 20 units using the associated angle

Reviewer: ```suggestion
        -- translate the point 20 units in the given direction (angle)
```
